tests/core/test_plan.py

```diff
@@ -215,6 +216,7 @@ def test_restate_models(sushi_context_pre_scheduling: Context):
     assert not plan.requires_backfill
 
 
+@pytest.mark.integration
```

Comment:
Should this one https://github.com/TobikoData/sqlmesh/blob/9ca41c82abcc93bab41935793338080883f721e7/tests/core/test_plan.py#L1239 be integration too?